src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -2762,4 +2764,16 @@ private function addJsonStreamerSection(ArrayNodeDefinition $rootNode, callable
             ->end()
         ;
     }
+
+    private function addObjectMapperSection(ArrayNodeDefinition $rootNode, callable $enableIfStandalone): void
+    {
+        $rootNode
+            ->children()
+                ->arrayNode('object_mapper')
```

Comment:
this is never IIUC, there's no corresponding readConfigEnabled call in FrameworkExtension

Reply:
Indeed, I removed the configuration node I don't think its needed.